commands_to_load/Reminders.py

```diff
@@ -152,9 +164,10 @@ def __init__(self, bot):
 						msg = await chan.get_message(cid_mid_dct['mid'])
 						ctx = await self.bot.get_context(msg)
 						if ctx.valid and helper_files.testenv.TestCog.check_test_environment(ctx):
-							fmt = '<@{0}> ```{1}```'
-							logger.info('[Reminders.py get_message()] sent off reminder to '+str(ctx.message.author)+" about \""+ctx.message.content+"\"")
-							await ctx.send(fmt.format(ctx.message.author.id, ctx.message.content))
+							fmt = '<@{0}>\n {1}'
+							logger.info('[Misc.py get_message()] sent off reminder to '+str(ctx.message.author)+" about \""+ctx.message.content+"\"")
+							eObj = embed(author=settings.BOT_NAME, avatar=settings.BOT_AVATAR, description=fmt.format(ctx.message.author.id, mssg), footer='Reminder')
```

Comment:
bad luck bud

2018-09-21 07:26:00 = INFO - [Misc.py get_message()] sent off reminder to modernNeo#6816 about ".remindmein 2 minutes to do the thing"
2018-09-21 07:26:00 = ERROR - [Reminders.py get_message()] Ignoring exception when generating reminder:
2018-09-21 07:26:00 = WARNING - Traceback (most recent call last):
2018-09-21 07:26:00 = WARNING -
2018-09-21 07:26:00 = WARNING -   File "/usr/src/app/commands_to_load/Reminders.py", line 169, in get_messages
.    eObj = embed(author=settings.BOT_NAME, avatar=settings.BOT_AVATAR, description=fmt.format(ctx.message.author.id, mssg), footer='Reminder')
2018-09-21 07:26:00 = WARNING -
2018-09-21 07:26:00 = WARNING - NameError: name 'mssg' is not defined
2018-09-21 07:26:00 = WARNING -